ignf_gpf_sdk/__main__.py

```diff
@@ -134,6 +134,16 @@ def parse_args(args: Optional[Sequence[str]] = None) -> argparse.Namespace:
         o_sub_parser.add_argument("--name", "-n", type=str, default=None, help="Nom du workflow à extraire")
         o_sub_parser.add_argument("--step", "-s", type=str, default=None, help="Étape du workflow à lancer")
         o_sub_parser.add_argument("--behavior", "-b", type=str, default=None, help="Action à effectuer si l'exécution de traitement existe déjà")
+        o_sub_parser.add_argument("--tag", "-t", type=str, nargs=2, action="append", metavar=("Clef", "Valeur"), default=[], help="tag à ajouter aux actions (plusieurs tags possible)")
+        o_sub_parser.add_argument(
+            "--commentaire",
```

Comment:
Pour que ce soit homogène.
```suggestion
            "--comment",
```